core/src/data/properties.cpp

```diff
@@ -31,14 +31,22 @@ void Properties::setSorted(std::vector<Item>&& _items) {
 const Value& Properties::get(const std::string& key) const {
     const static Value NOT_FOUND(none_type{});
 
-    const auto it = std::lower_bound(props.begin(), props.end(), key,
-                                     [](const auto& item, const auto& key) {
-                                         return keyComparator(item.key, key);
-                                     });
-
-    if (it == props.end() || it->key != key) {
+    const auto it = std::find_if(props.begin(), props.end(),
```

Comment:
Were these tests with Release builds and on std::vector? In my tests find_if was faster or just as fast than manual iteration. This is really a thing to let the compiler can figure out :)